src/EditorFeatures/CSharpTest2/Recommendations/RecommenderTests.cs

```diff
@@ -82,8 +82,8 @@ private Task CheckResultAsync(string text, int position, bool absent, CSharpPars
                 }
                 else
                 {
-                    var result = (await RecommendKeywordsAsync(position, context)).Single();
-                    Assert.NotNull(result);
+                    var result = (await RecommendKeywordsAsync(position, context))?.Single();
```

Comment:
❓ Why would you need `?.` here?

:bulb: Consider the following:

```csharp
var result = await RecommendKeywordsAsync(position, context);
Assert.Single(result);
```

Reply:
Good catch. The `?.` was unnecessary. Removed it

Having a message in the assert is what I'm after, so I'm leaving it. That improves the experience working in IDE codebase. Below is the generic message assert message the old code would produce (not very informative):

```
Test 'Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Recommendations.WhenKeywordRecommenderTests.TestInCasePattern' failed:
	System.ArgumentNullException : Value cannot be null.
Parameter name: source
	at System.Linq.Enumerable.Single[TSource](IEnumerable`1 source)
	at Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Recommendations.RecommenderTests.<CheckResultAsync>d__5.MoveNext()
...
```